datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -138,6 +138,8 @@ def execute(self):
                         continue
                     elif column_type == 'tag':
                         tags.append(transformer(None, value))
+                    elif column_type == 'tag_list':
+                        tags.extend(transformer(None, value))
```

Comment:
I'll submit a separate PR to `datadog_checks_base` to implement this new column type, just wanted to draft and make sure it would make sense and fit the use case here. :-)

Reply:
https://github.com/DataDog/integrations-core/pull/8147